bin/dotd

```diff
@@ -291,27 +291,20 @@ def do_dtt_subroutine
   print_dtt_message
   should_i 'DTT' do
     view_commits base: 'test', head: 'staging'
```

Comment:
One can make that argument.

My feeling is that we're never actually going to change our decision based on what we see in the commits. Rather, I like to see the commits just to know the approximate complexity of what I have DTT'd.

Would be interested in knowing how others feel.